test/Klix/Widget/CheckoutWidgetTest.php

```diff
@@ -22,6 +22,7 @@ public function testHtmlAttributesWidgetHtmlGeneration() {
 		self::assertEquals("klix-checkout", $xml->getName());
 		self::assertEquals($widgetConfiguration->getWidgetId(), $xml->attributes()["widget-id"]);
 		self::assertEquals($widgetConfiguration->getLanguage(), $xml->attributes()["language"]);
+		self::assertEquals($widgetConfiguration->getBackToMerchantUrl(), $xml->attributes()["back-to-merchant-url"]);
```

Comment:
There are no changes in CheckoutWidget class, therefore, both new URLs will not be included in the widget HTML representation

Reply:
BTW, have you launched all existing unit tests? This test does not fail because null is equal to null (back to merchant URL is not set in a test)